content/en/docs/tutorials/acme/http-validation.md

```diff
@@ -116,8 +116,9 @@ control how cert-manager interacts with Ingress resources:
 - If the `ingress` field is specified, then an Ingress resource with the same
   name in the same namespace as the Certificate must already exist and it will
   be modified only to add the appropriate rules to solve the challenge.
-  This field is useful for the GCLB ingress controller, as well as a number of
-  others, that assign a single public IP address for each ingress resource.
+  This field is useful for the Google Cloud Loadbalancer ingress controller, 
+  as well as a number of others, that assign a single public IP address for 
+  each ingress resource.
```

Comment:
I personally think that for someone reading it for the first time, this field can be overlooked between other optional `fields`. Since, it's a **required** step for GCP, I suppose `This field is useful for` -> `This field is **required** for...`
Also, a note that some cloud providers like GCP require an existing ingress should help as well, maybe:


Google Cloud Loadbalancer ingress controller (as well as a number of others which assign single public IP address for each ingress resource) **require** `ingress` name to be specified this field. When the `ingress` field is specified, an Ingress resource with the same name in the same namespace as the Certificate must already exist and it will be modified only to add the appropriate rules to solve the challenge. After the challenge is complete and the certificate is issued, the rules created for solving the challenge will be removed. 

What do you think about this change?

Reply:
It is not per say a requirement as if you use the ingress-shim with the `http01-edit-in-place` annotation it is not applied